src/dotnet-serve/.netconfig

```diff
@@ -0,0 +1,6 @@
+[serve]
+	header = x-frame-options: SAMEORIGIN
+	mime = .netconfig=text/plain
```

Comment:
Can you provide an example that has multiple items? What's the delimiter?

Reply:
Multi-valued variables are supported built-in, just like in git-config, with multiple entries. No need for separators:

```
  mime = .netconfig=text/plain
  mime = .cs=text.plain
  mime = ....
```